SECURITY.md

```diff
@@ -62,7 +62,7 @@ In Maven, you can use the [os-maven-plugin](https://github.com/trustin/os-maven-
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>1.1.33.Fork26</version>
+      <version>2.0.3.Final</version>
```

Comment:
This is a cherry-pick. Making real changes in a cherry-pick is not a great idea. I wasn't "changing it anyways", in that I didn't even open up an editor for the file.